content/docs/quickstart/cpp.md

```diff
@@ -9,63 +9,111 @@ description: This guide gets you started with gRPC in C++ with a simple working
 
 #### gRPC
 
-To install gRPC on your system, follow the [instructions to install gRPC C++ via make](https://github.com/grpc/grpc/blob/master/src/cpp/README.md#make).
+Follow the gRPC installation instructions for your OS:
 
-To run the example code, please ensure `pkg-config` is installed on your
-machine before you build and install gRPC in the previous step, since the
-example `Makefile`s try to look up the installed gRPC path using `pkg-config`.
-On Debian-based systems like Ubuntu, this can usually be done via
-`sudo apt-get install pkg-config`.
+- macOS:
+
+  ```sh
+  $ brew install grpc
+  ```
+
+- Any OS: to build and install from sources, refer to [Building from
```

Comment:
A better entry point for installing gRPC C++ from the users's perspective (as opposed to being a gRPC contributor) is actually https://github.com/grpc/grpc/tree/master/src/cpp#to-start-using-grpc-c (which then points to building.md for more details).

Reply:
Done